egs/wsj/s5/steps/nnet3/xconfig_to_configs.py

```diff
@@ -219,17 +227,20 @@ def write_config_files(config_dir, all_layers):
             raise
 
 
-def add_nnet_context_info(config_dir, nnet_edits=None):
+def add_nnet_context_info(config_dir, existing_model=None,
+                                edits_config=None):
     """Create the 'vars' file that specifies model_left_context, etc."""
 
-    common_lib.execute_command("nnet3-init {0}/ref.config "
-                               "{0}/ref.raw".format(config_dir))
     model = "{0}/ref.raw".format(config_dir)
-    if nnet_edits is not None:
-        model = "nnet3-copy --edits='{0}' {1} - |".format(nnet_edits,
-                                                          model)
-    out = common_lib.get_command_stdout('nnet3-info "{0}" | head -n 4 '
-                                        .format(model))
+    if edits_config is not None:
+        model = """ - | nnet3-copy --edits-config={0} - {1}""".format(edits_config,
+                                                              model)
+    common_lib.execute_command("""nnet3-init {0} {1}/ref.config """
+                                 """ {2} """.format(existing_model if
```

Comment:
Thanks for checking this, Vimal.
I'd like it to conform to Google standards as much as possible, but don't be too strict with the column-length of 80, it's hard to stick to that  in Python, what with the indentation.